tools/serviceTest.sh

```diff
@@ -1,11 +1,8 @@
 #!/bin/bash
 
 ########################### TESTING ############################
-# For now on Mac:
-# docker logs -f `docker ps -q --filter name=$SERVICE_NAME`
-#
-# For now on Linux:
-# sudo tail -f /var/log/syslog
+# For Linux and Mac:
+# hzn service log -f $SERVICE_NAME
```

Comment:
Not all users will have a recent enough version of `hzn` that it will have `hzn service log`. Please update the script to check that in a bash function before using it. Thx.